libs/order/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@daffodil/order",
+  "version": "0.0.0-PLACEHOLDER",
+  "description": "A package for interacting with platforms that provide geographic data information like States and Countries",
```

Comment:
`A client-side Angular package for managing ecommerce orders agnostic to the ecommerce platform.`

Reply:
changedc